src/Components/App/index.js

```diff
@@ -24,3 +22,34 @@ export default class App extends Component {
     );
   }
 }
+
+
+export function createTable(firstRowTable, query) {
```

Comment:
Deze hele functie schreeuwt anti react ofzo.. Je bent hier gewoon puur javascript aan het schrijven om problemen op te lossen die in React met veel simpelere syntax zijn te doen :s.. Ik weet niet zo goed waar ik moet beginnen, maar het is de bedoeling dat al onze html uit JSX en react componenten voort komt.

Dus maak er een React class component of function component van, en laat deze JSX returnen in de render

Reply:
Ok, got one, i think :P